gce/util.sh

```diff
@@ -9,6 +9,171 @@
 #	util::remote_cmd
 
 
+
+# Helpers #
+# These functions implement lower level operations required for deploying
+# and destroying Google Compute Engine instances.  These functions should be
+# considered private, scoped to the provider.
+
+function __init_network() {
+	local fw_rule_allow_all="$GCP_USER-gluster-kubernetes-allow-all"
+	echo "-- Checking for network $GCP_NETWORK"
+	if ! gcloud compute networks describe "$GCP_NETWORK" &>/dev/null; then
+		echo "-- Network not found. Creating network now."
+		gcloud compute networks create "$GCP_NETWORK" --mode=auto || exit 1
```

Comment:
decide if you want functions in a util library to exit vs. return 1. I think the later is better.

Reply:
agree, will go through and correct this.